### PR TITLE
Fixes #12934 - Unhiding paramter values does not clear their value

### PR DESCRIPTION
**Pull request by orrabin**

### CODE REVIEW

File: app/assets/javascripts/lookup_keys.js [hunk 1]
```diff
@@ -300,7 +300,7 @@ function toggle_value_hidden(target, shown){
   if (shown) {
     $(target).replaceWith($('<input/>').attr(_.extend({ type: 'password' }, attrs)));
   } else {
-    $(target).replaceWith($('<textarea/>').attr(_.extend({ placeholder: _('Value'), rows: 1 }, attrs)));
+    $(target).replaceWith($('<textarea/>').attr(_.extend({ placeholder: _('Value'), rows: 1 }, attrs)).val(attrs['value']));
```
Comment (tbrisker) [inline app/assets/javascripts/lookup_keys.js]: while you are here, target is already a jQuery object, no need to wrap it in `$()`.

**Comment by tbrisker** [comment]
:+1: LGTM.

**Comment by orrabin** [comment]
@tbrisker I fixed your comment

**Comment by dLobatog** [comment]
@orrabin @tbrisker Thanks, I'll merge but please include a test next time, it's hard to notice when things like this break.

**Comment by dLobatog** [comment]
Merged as 947393d8cb0c6ddb452cbc86204f78259022a9d3, thanks @orrabin!

**Pull request closed without merging**